tgui/packages/tgui/interfaces/CrewConsoleSkyrat.js

```diff
@@ -12,20 +12,19 @@ const HEALTH_COLOR_BY_LEVEL = [
   '#e74c3c',
   '#801308',
 ];
-const HEALTH_ICON_BY_LEVEL = [
-  'heart',
-  'heart',
-  'heart',
-  'heart',
-  'heartbeat',
-  'skull',
-];
+
+const STAT_LIVING = 0;
+const STAT_DEAD = 4;
+
 const jobIsHead = (jobId) => jobId % 10 === 0;
 
 const jobToColor = (jobId) => {
   if (jobId === 0) {
     return COLORS.department.captain;
   }
+  if (jobId >= 200 && jobId < 240) {
+    return COLORS.department.centcom;
+  }
```

Comment:
How does this make a difference, to move it up here?

Reply:
I actually forgot to move that back down - it's going to be to put ERT and what not at the top but decided to save that for the enhancement